user-service/pom.xml

```diff
@@ -28,7 +28,6 @@
     <artifactId>user-service</artifactId>
 
     <properties>
-        <spring-cloud-starter-ribbon.version>1.4.7.RELEASE</spring-cloud-starter-ribbon.version>
         <swagger.version>1.5.21</swagger.version>
```

Comment:
Are we still using swagger at this point?

Reply:
I don't believe we are using Swagger anymore so this could be removed along with the dependency